content/v1/features/unity.md

```diff
@@ -465,4 +465,30 @@ The user can also set the volume limit for all the nodes in the cluster by speci
 ## Log Levels
 The CSI Driver for Dell EMC Unity allows users to configure different log levels using **logLevel** parameter. 
 
-The **logLevel** parameter needs to be configured in unity-creds secret created from secret.json or secret.yaml. The supported log levels are Info/Debug/Warn/Error. The default level is Info if logLevel is not configured by the user. The parameter can be changed dynamically without the need of driver re-installation or restart.
\ No newline at end of file
+The **logLevel** parameter needs to be configured in unity-creds secret created from secret.json or secret.yaml. The supported log levels are Info/Debug/Warn/Error. The default level is Info if logLevel is not configured by the user. The parameter can be changed dynamically without the need of driver re-installation or restart.
+
+## Single Pod Access Mode for PersistentVolumes
+Starting in version 2.1, CSI Driver for Unity now supports a new accessmode `ReadWriteOncePod` for PersistentVolumes and PersistentVolumeClaims. With this feature, CSI Driver for Unity allows to restrict volume access to a single pod in the cluster
```

Comment:
Line 471 should read "...the CSI Driver for Unity now supports..."

"...the CSI driver for allos..."